DockerOrchestration/TendermintCore/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:1.8
```

Comment:
I'll switch to that. Before easter I wasn't sure whether those were actively maintained :)